app/lib/event_bus.rb

```diff
@@ -28,8 +28,16 @@ def self.reset!
 # wiring
 
 # automation
-[:message_thread_created, :message_created, :message_draft_submitted].each do |event|
-  EventBus.subscribe_job event, Automation::ApplyRulesForEventJob
+EventBus.subscribe :message_thread_created, ->(message_thread, queue_name = :medium_priority) do
```

Comment:
Tu trosku pozor na domenu. event bus by nemal tusit nic o "frontach" mozeme to nazvat skor `event_priority` alebo proste `priority` a vnutri sa to prelozi na to iste. (nitpick)

Zaroven by som to nevolal medium priority ale `default`

Reply:
Upravila som, ale len sa to posunulo trochu nizsie, queue stale spominame v event bus.